shipyard-controller/handler/projecthandler_test.go

```diff
@@ -66,6 +67,11 @@ func TestGetAllProjects(t *testing.T) {
 				EventSender:           &fake.IEventSenderMock{},
 				EnvConfig:             config.EnvConfig{ProjectNameMaxSize: 200},
 				RepositoryProvisioner: &fake.IRepositoryProvisionerMock{},
+				RemoteURLValidator: fake.RequestValidatorMock{
+					ValidateFunc: func(url string) error {
+						return nil
+					},
+				},
```

Comment:
since it is repeated over and over, would it make sense to have it defined only once?

Reply:
we can, but we define everything like this, it's not a deal, I will rewrite it